ci/bamboo/build-manylinux-release.sh

```diff
@@ -20,17 +20,27 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-# Build nupic.bindings python extension using manylinux docker image.
```

Comment:
The git relationship `bootstrap-manylinux-prototype.sh → build-manylinux-release.sh` is incorrect. This is an entirely new script.